### PR TITLE
Fix bug values for variables

### PR DESCRIPTION
**Pull request by nmartinnez**
the default values of the variables were corrected

### CODE REVIEW

File: main.tf [hunk 1]
```diff
@@ -30,10 +30,13 @@ resource "aws_cognito_user_pool" "pool-1" {
...
   mfa_configuration             = "${var.mfa_configuration}"
-  sms_configuration             = "${var.sms_configuration}"
+  sms_configuration             = {
+    sns_caller_arn              = "${var.sns_caller_arn}"
+    external_id                 = "${var.external_id}"
+  }
   sms_authentication_message    = "${var.sms_authentication_message}"
   user_pool_add_ons             = "${var.user_pool_add_ons}"
-  verification_message_template = "${var.verification_message_template}"
+  #verification_message_template = "${var.verification_message_template}"
```
Comment (pazmarianoj) [inline main.tf:39]: Is this an error or we don't want that input?

File: variables.tf [hunk 2]
```diff
@@ -276,6 +276,7 @@ variable "require_symbols" {
 variable "require_uppercase" {
   type        = "string"
   description = "Whether you have required users to use at least one uppercase letter in their password"
+  default     = ""
```
Comment (pazmarianoj) [inline variables.tf]: Can we set a default value? I would vote for False, but I don't know which AWS puts in here.

**Review by pazmarianoj — COMMENTED** [review]
Added my comments in line

File: main.tf (hunk 1 above)
Comment (nmartinnez) [reply main.tf:39]: this entry is not necessary, but is commented on by a test

File: variables.tf (hunk 2 above)
Comment (nmartinnez) [reply variables.tf]: That variable was as required by a typing error, true or false can be passed as a value if it is used

**Review by pazmarianoj — APPROVED** [review]
+1

**Pull request merged**